### PR TITLE
Test for varLib mutator

### PR DESCRIPTION
**Pull request by miguelsousa**
The test fails on py3 with the traceback below. Not quite sure what needs fixing. Help please.

```python
Traceback (most recent call last):
  File "mutator_test.py", line 104, in test_varlib_mutator_ttf
    mutator(args)
  File "fonttools/Lib/fontTools/varLib/mutator.py", line 57, in main
    varfont.save(outfile)
  File "fonttools/Lib/fontTools/ttLib/__init__.py", line 219, in save
    self._writeTable(tag, writer, done)
  File "fonttools/Lib/fontTools/ttLib/__init__.py", line 658, in _writeTable
    tabledata = self.getTableData(tag)
  File "fonttools/Lib/fontTools/ttLib/__init__.py", line 669, in getTableData
    return self.tables[tag].compile(self)
  File "fonttools/Lib/fontTools/ttLib/tables/_h_h_e_a.py", line 44, in compile
    return sstruct.pack(hheaFormat, self)
  File "fonttools/Lib/fontTools/misc/sstruct.py", line 75, in pack
    data = struct.pack(*(formatstring,) + tuple(elements))
struct.error: required argument is not an integer
```

### CODE REVIEW

**Comment by anthrotype** [comment]
the width and lsb in hmtx must be rounded to integers; if they are float, the struct module will raise struct.error on Python 3. On Python 2, unfortunately, it truncates the float...

```diff
diff --git a/Lib/fontTools/varLib/__init__.py b/Lib/fontTools/varLib/__init__.py
index 8f2d7fc..eead87f 100644
--- a/Lib/fontTools/varLib/__init__.py
+++ b/Lib/fontTools/varLib/__init__.py
@@ -154,8 +154,8 @@ def _SetCoordinates(font, glyphName, coord):
 
        glyph.recalcBounds(glyf)
 
-       horizontalAdvanceWidth = rightSideX - leftSideX
-       leftSideBearing = glyph.xMin - leftSideX
+       horizontalAdvanceWidth = round(rightSideX - leftSideX)
+       leftSideBearing = round(glyph.xMin - leftSideX)
        # XXX Handle vertical
        font["hmtx"].metrics[glyphName] = horizontalAdvanceWidth, leftSideBearing
 
diff --git a/Tests/varLib/data/MutatorTest.ttx b/Tests/varLib/data/MutatorTest.ttx
index e5786f5..2467eb3 100644
--- a/Tests/varLib/data/MutatorTest.ttx
+++ b/Tests/varLib/data/MutatorTest.ttx
@@ -16,9 +16,9 @@
     <ascent value="918"/>
     <descent value="-335"/>
     <lineGap value="0"/>
-    <advanceWidthMax value="666"/>
+    <advanceWidthMax value="667"/>
     <minLeftSideBearing value="7"/>
-    <minRightSideBearing value="3"/>
+    <minRightSideBearing value="4"/>
     <xMaxExtent value="656"/>
     <caretSlopeRise value="1"/>
     <caretSlopeRun value="0"/>
```

File: Lib/fontTools/varLib/mutator.py [hunk 1]
```diff
@@ -48,8 +48,8 @@ def main(args=None):
 			coordinates += GlyphCoordinates(var.coordinates) * scalar
 		_SetCoordinates(varfont, glyphname, coordinates)
 
-	print("Removing GX tables")
-	for tag in ('fvar','avar','gvar'):
+	print("Removing variable tables")
+	for tag in ('fvar','avar','gvar','HVAR'):
```
Comment (behdad) [inline Lib/fontTools/varLib/mutator.py]: Drop all varfonts tables?

I like to pick up completing mutator soon.

Comment (miguelsousa) [inline Lib/fontTools/varLib/mutator.py]: So leave it as it was is what you're saying, yes? Sorry I didn't know this was a WIP.
What's missing, BTW?

**Comment by miguelsousa** [comment]
Thanks Cosimo. I'll apply those changes in a bit.

**Comment by miguelsousa** [comment]
All done, I think.

File: Lib/fontTools/varLib/mutator.py (hunk 1 above)
Comment (behdad) [inline Lib/fontTools/varLib/mutator.py]: I meant, you can go ahead and drop VVAR, STAT, MVAR, etc as well. And yes, mutator is very much a WIP. But then again, so is the rest of varLib. But mutator is the least complete one.

Comment (miguelsousa) [inline Lib/fontTools/varLib/mutator.py]: Got it.

**Pull request merged**